src/main/java/org/apache/commons/collections4/list/AbstractLinkedList.java

```diff
@@ -514,6 +515,8 @@ protected void addNodeAfter(final Node<E> node, final E value) {
      * @throws NullPointerException if either node is null
      */
     protected void addNode(final Node<E> nodeToInsert, final Node<E> insertBeforeNode) {
+        Objects.requireNonNull(nodeToInsert,"The new node on insert must not be null.");
+        Objects.requireNonNull(insertBeforeNode,"The node to insert before must not be null.");
```

Comment:
Please add a space after the comma.

Reply:
Thanks for your suggestions, I will fix the commit right away